spec/menu_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'menu'
+require 'takeaway'
+
+describe Menu do
```

Comment:
Unit tests should be grouped by method name to make them easier to read. Otherwise, testing is good. Could include edge cases to go further.